x/leveragelp/types/codec.go

```diff
@@ -35,6 +36,7 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		&MsgWhitelist{},
 		&MsgDewhitelist{},
 		&MsgClaimRewards{},
+		&MsgClaimAllRewards{},
```

Comment:
register the 2 txs

Reply:
MsgClaimAllRewards tx is already implemented in leveragelp